src/AppBundle/Controller/AdminController.php

```diff
@@ -75,45 +75,47 @@ public function supprimerPostAction($id, Request $request)
 		  'form'   => $form->createView()
 		));
     }
-	
+
 	/**
-     * @Route("/supprimer_comment/{id}", name="supprimer_comment")	 
-     */	
-	public function supprimerCommentAction($id, Request $request)           
-    {			
+     * @Route("/supprimer_comment/{id}", name="supprimer_comment")
+     */
+	public function supprimerCommentAction($id, Request $request)
+    {
 	   /**Recupere l'EntityManager $em */
 	   $em = $this->getDoctrine()->getManager();
 	   /**Recupere le repository */
 	   $repository = $em->getRepository('AppBundle:Comment');
-	   
+
 	    /**Recupere l'entité qui correspond à l'id */
 	   $comment = $repository->find($id);
-	   
+
 	   /**On supprime le post */
 	   $em->remove($comment);
 	   $em->flush();
-	   
+
 	   $request->getSession()->getFlashBag()->add('notice', 'Le commentaire a été bien supprimé');
-		
-	/**
-	 * Ici je prefere rediriger vers l'article, mais j'arrive pas	 
-	 */
+
+		/**
+		 * Ici je prefere rediriger vers l'article, mais j'arrive pas
+		 */
        return $this->redirect($this->generateUrl('blog'));
     }
+
 	/**
-     * Cela va me servir pour recuperer le formulaire et non repeter 
+     * Cela va me servir pour recuperer le formulaire et non repeter
 	 * la même chose tjs. dans les autres methodes.
      */
 	protected function handlePostForm(Post $post, Request $request)
 	{
-		$form = $this->createForm(new Posttype(), $post);
-		
+		$form = $this->createForm(new PostType(), $post);
+
 		if ($form->handleRequest($request)->isValid())
 		{
-			 $em = $this->getDoctrine()->getManager();
+			$post->setAuteur($this->getUser());
```

Comment:
`$this->getUser()` dans un contrôleur permet de récupérer l'utilisateur connecté, et si l'utilisateur n'est pas connecté, cette fonction renvoie `null`